content/lessons/solid/solid-part-2.md

```diff
@@ -0,0 +1,14 @@
+---
+title: SOLID Principles Part 2
+template: lesson
+draft: false
+slug: /courses/Solid
+course: SOLID
+tags:
+- Usability
+- Maintainability
+- OOP
+description: "Put your new SOLID knowledge to the test as you rewrite principle-violating code so it is in line with the principles."
```

Comment:
I would add that the coding will be done in Java, just so people have that idea going into it.